xbmc/filesystem/AddonsDirectory.cpp

```diff
@@ -62,12 +62,27 @@ const std::array<TYPE, 5> infoProviderTypes = {
   ADDON_SCRAPER_TVSHOWS,
 };
 
+const std::vector<TYPE> lookAndFeelTypes = {
```

Comment:
Is there any reason why this isn't an `std::set<TYPE>`? It will always contain unique TYPE entries and it would make looking for a specific TYPE faster.